### PR TITLE
[TW15140277] Steps + Flowmap (Timeline) + Hero Jump Nav

### PR DESCRIPTION
**Pull request by jacine**
Also includes:

- Updated Pattern Lab
- Adjustments to  Switchboard due to Pattern Lab changes.

### CODE REVIEW

**Comment by jacine** [comment]
Weird, but Pantheon multidev hook did not report back here... It exists though.  
http://pr-68-jcc-srl.chapterthree.com/debt-collection/general-information

File: web/themes/custom/atrium/templates/paragraph/paragraph--step-container.html.twig [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+    * used to render steps.
+    *
+    * 1. Vertical Steps (vertical_steps): Known as "Steps" in Pattern Lab.
+    * 2. Flowmap (flowmap): Known as "Timeline" in Pattern Lab.
+    */
+    #}
+    {% set component = paragraph.field_step_display.value %}
+
+    {% set headergroup = {
+      brow: (component == 'vertical_steps') ? 'Next steps'|t : null,
```
Comment (zakiya) [inline web/themes/custom/atrium/templates/paragraph/paragraph--step-container.html.twig:27]: "Next Steps" will need to be configurable in Drupal. (It wasn't in the wireframes for Steps or Flowmaps.) I'll make a task to add field_brow to the Step Container Paragraph Type.

File: web/themes/custom/atrium/templates/views/views-view.html.twig [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+ *   view content.
+ * - attachment_after: An optional attachment view to be displayed after the
+ *   view content.
+ * - dom_id: Unique id for every view being printed to give unique class for
+ *   Javascript.
+ *
+ * @see template_preprocess_views_view()
+ */
+#}
+{% set tag = tag|default('div') %}
```
Comment (zakiya) [inline web/themes/custom/atrium/templates/views/views-view.html.twig:33]: 🙂

File: web/themes/custom/atrium/templates/paragraph/paragraph--step-container.html.twig (hunk 1 above)
Comment (jacine) [reply web/themes/custom/atrium/templates/paragraph/paragraph--step-container.html.twig:27]: 👍

**Pull request merged**